chain/polkadot/parachain_node.go

```diff
@@ -0,0 +1,274 @@
+package polkadot
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/avast/retry-go/v4"
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	p2pCrypto "github.com/libp2p/go-libp2p-core/crypto"
```

Comment:
Import names should be all lowercase.

Reply:
You'd think a linter somewhere should catch that.

Reply:
Yeah, you'd think so. The blog post on this is 7.5 years old, so there's been plenty of time for someone to write one: https://go.dev/blog/package-names

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns...

Reply:
There may be a linter with golangci-lint we need to activate. There's so many of them; hard to keep track.